themes/default/content/templates/kubernetes/aws/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Kubernetes Cluster on AWS"
+layout: template
+
+# Make sure this is description accurate for this template.
+meta_desc: The AWS Kubernetes template makes it easy to deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS.
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS.
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: kubernetes-aws
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Amazon Web Services
+  slug: aws
+
+# Be sure to replace this image. Figma source file:
+# https://www.figma.com/file/lGrSpwbGGmbixEuewMbtkh/Template-Architecture-Diagrams?node-id=15%3A196
+meta_image: meta.png
+
+# The content below is meant help you get started and to serve as a guide to work by. Feel free to adjust it needed for your template.
+---
+
+The AWS Kubernetes template deploys a managed Kubernetes cluster on AWS with Pulumi infrastructure as code. It deploys an [Amazon EKS cluster]({{< relref "/registry/packages/eks/api-docs/cluster" >}}) that provides a managed Kubernetes control plane and nodes. The nodes run on an [Amazon EC2 instance]({{< relref "/registry/packages/awsx/api-docs/ec2" >}}) connected to a private subnet in a [VPC]({{< relref "/registry/packages/awsx/api-docs/ec2/vpc" >}}). The template also deploys Internet-facing load balancers connected to public subnets. The template generates a complete infrastructure as code program to give you a working project out of the box that you can customize easily and extend to suit your needs.
+
+![An architecture diagram of the Pulumi AWS Kubernetes template](./architecture.png)
+
+## Using this template
+
+To use this template to deploy your own managed Kubernetes cluster, make sure you've [installed Pulumi]({{< relref "/docs/get-started/install" >}}) and [configured your AWS credentials]({{< relref "/registry/packages/aws/installation-configuration#credentials" >}}), then create a new [project]({{< relref "/docs/intro/concepts/project" >}}) using the template in your language of choice:
+
+{{% chooser language "typescript,python,go,csharp,yaml" / %}}
```

Comment:
C# isn't ready yet, so we may want to pull this part out (and then add it back when the C# template is ready).